handlers/admin/data-contributions.js

```diff
@@ -16,23 +20,30 @@ function editDataContribution(request, reply) {
       request.yar.flash('success', 'Data Contribution updated successfully.');
       return reply.redirect('/admin/data-contributions');
     })
-    .catch(() => {
-      // FIXME: Log error
+    .catch((error) => {
+      console.error(error.detail); // eslint-disable-line no-console
```

Comment:
yes, just to iterate on what @vitorbaptista said - STDOUT **is** our "standardised logging" interface, @nightsh . As we use docker, our app environments are stateless, and we don't rely on files or data co-located with the app. So for logs, we pipe the stream to papertrail.